contrib/bentoml/deployment_from_bento.yaml

```diff
@@ -0,0 +1,97 @@
+apiVersion: resources.yatai.ai/v1alpha1
+kind: Bento
+metadata:
+  name: fraud-detection
+  namespace: kubeflow
+spec:
+  image: docker.io/bentoml/fraud_detection:o5smnagbncigycvj
+  runners:
+  - name: ieee-fraud-detection-0
+    runnableType: XGBoost
+  - name: ieee-fraud-detection-1
+    runnableType: XGBoost
+  - name: ieee-fraud-detection-2
```

Comment:
nit, not a blocker. does it need three runners? kubeflow itself is pretty resource heavy so wondering if lightweight would be better

Reply:
Using multiple runners is part of the demonstration of BentoML capabilities. Each pod only requests 0.1 CPU cores so hopefully resource is not a concern.